portfolio/src/main/java/com/google/sps/servlets/DeleteServlet.java

```diff
@@ -0,0 +1,37 @@
+package com.google.sps.servlets;
+
+import com.google.sps.data.Comment;
+import java.util.Date;
+import com.google.gson.Gson;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+
+/** Servlet that deletes some data. */
+@WebServlet("/delete-data")
+public class DeleteServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+      
+      Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
+      DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+      PreparedQuery results = datastore.prepare(query);
+
+      ArrayList<Entity> entities = new ArrayList<>();
+      for (Entity entity : results.asIterable()) {
+          datastore.delete(entity.getKey());
+      }
+      response.getWriter().print("empty response");
+  }
```

Comment:
Did the training course say to make a new servlet for this? Because I would have thought this could have been done with something like a doDelete method and using the DELETE verb. But maybe they wanted you to practice managing multiple servlets. Or maybe there is some convention that I'm not aware of saying POST is better to use than DELETE

Reply:
Yeah, they told us to make a new servlet